policy/modules/services/xserver.te

```diff
@@ -936,6 +936,7 @@ tunable_policy(`xdm_bind_vnc_tcp_port',`
 
 optional_policy(`
 	accountsd_read_lib_files(xdm_t)
+        watch_dirs_pattern(xdm_t, accountsd_var_lib_t, accountsd_var_lib_t)
```

Comment:
This has been addressed with commit 478c5f3df1ffed004dbefb8c33449bd1444bf634.
Note an interface should be used when accessing types outside the module.